pkg/ccl/changefeedccl/changefeed_stmt.go

```diff
@@ -1326,6 +1328,21 @@ func logChangefeedFailedTelemetry(
 	log.StructuredEvent(ctx, changefeedFailedEvent)
 }
 
+// logChangefeedRetryableErrorTelemetry logs a retryable error for enterprise changefeeds.
+func logChangefeedRetryableErrorTelemetry(ctx context.Context, job *jobs.Job, err error) {
+	var changefeedEventDetails eventpb.CommonChangefeedEventDetails
```

Comment:
Does this need a `job != nil` check like the other code?  If not could just do `changefeedEventDetails := ...`